swift.en.md

```diff
@@ -0,0 +1,340 @@
+# Swift coding style
+
+## Introduction
+
+This document defines the conventions for writing Swift code at Cookpad Inc.
+
+This guide directed for Swift 3.0.2.
+
+## Objective
+
+This guides aim to accomplish these objectives.
+
+1. Clarify purposes 
+1. Improve readability
+1. Keep consistency
+1. Reduce side-effects
+
+## Code style
+
+- **[MUST]** Use four spaces for 1-level of indent.  Do not use the horizontal tab character.
```

Comment:
**[MUST]** Use four spaces for 1-level of indentation.  Do not use the horizontal tab character.

OR EVEN BETTER

**[MUST]** Use two spaces for 1-level of indentation.  Do not use the horizontal tab character.